lua/telescope/pickers.lua

```diff
@@ -997,10 +971,11 @@ function Picker:get_status_updater(prompt_win, prompt_bufnr)
       return
     end
 
-    local prompt_len = #current_prompt
+    local text = self:get_status_text(opts)
+    local prompt_len = #self.prompt_prefix + #current_prompt
 
-    local padding = string.rep(" ", vim.api.nvim_win_get_width(prompt_win) - prompt_len - #text - 3)
-    vim.api.nvim_buf_clear_namespace(prompt_bufnr, ns_telescope_prompt, 0, 1)
+    local padding = string.rep(" ", vim.api.nvim_win_get_width(prompt_win) - prompt_len - #text)
```

Comment:
Dont remember why we had a `- 3` here but right now the max count is "out of screen"